### ISSUE TITLE
`prefer-spread` should not report on optional chaining

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#2298]
```js
scopeManager?.scopes.slice()
```

**Comment by fisker** [comment]
Note: Add test for `(scopeManager?.scopes).slice()` when work on it.

**Comment by MichaelBlm** [comment]
Can I attempt to fix this issue?
